scripts/lockfile-generators/download-npm.sh

```
@@ -254,25 +254,24 @@ echo ""
-            echo "[$count/$total] FAIL  Failed: $download_url" >&2
-            failed=$((failed + 1))
-            # Clean up partial download
+            echo "FAIL  Failed: $download_url" >&2
             rm -f "$DEST_DIR/$filename"
         fi
     fi
-done <<< "$refs"
+}
+export -f download_file
+
+echo "$refs" | xargs -n 2 -P 10 bash -c 'download_file "$1" "$2"' _
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)